src/main/java/com/hackathon03/codeguru/utils/HardCodedCredentials.java

```diff
@@ -0,0 +1,23 @@
+package com.hackathon03.codeguru.utils;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+public class HardCodedCredentials {
+
+    public void createSqlConnectionNoncompliant() throws Exception {
+        // Noncompliant: password is hardcoded.
+        final Connection connection = DriverManager.getConnection("some url",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We detected a hardcoded database connection password in your code. This could allow anyone with access to your server to perform privileged actions on the database. We recommend that you avoid hardcoded passwords when connecting to a database and instead use environment variables or a configuration file to set the password.

[Learn more](https://cwe.mitre.org/data/definitions/259.html)